_posts/2018-10-28-Hackathons.md

```diff
@@ -17,12 +17,11 @@ categories:
 
 |Hackathon|	Location|	Date	| Application deadline | Notes
 |:--------| :---    | :---      | :---                 | :---
-|[PearlHacks](https://pearlhacks.com/)| UNC Chapel Hill, Chapel Hill, NC | Feb 21-23, 2020 | N/A | For Female and Non-binary only. Great for First Timers. ~About 5.5 hours away from GGC
-|[Hacklytics](https://hacklytics.io/)| Georgia Tech, Atlanta, GA | Feb 22-23, 2020 | Feb 1, 2020 | Data Science themed Hackathon.
-|[DooleyHacks](https://dooleyhacks.com/) | Emory University, Atlanta, GA | Feb 22, 2020 | Feb 15, 2020 | 24 hour Hackathon.
-|[Horizons](https://horizons.hack.gt/) | Georgia Tech, Atlanta, GA | Feb 28-29, 2020 | TBA | Art & Tech Hackathon. Registration is open
-|[SouthernHacks](http://www.southernhacks.org/#home) | Georgia Southern University, Statesboro, GA | Feb 28-29, 2020 | TBA |~About 3.5 hours away from GGC. Registration is open
 |[DataFest](https://www.engage.emory.edu/s/1705/alumni/index3.aspx?sid=1705&gid=3&pgid=11397&content_id=9979) | Emory Universtiy, Atlanta, GA | March 20-22, 2020 | TBA | Registration is open.
+|[MiamiHacks](https://miamihacks.com/)| Florida International University, Miami, FL | March 20-22, 2020 | TBA | Registration is open. ~About 10 hours away from GGC.
+|[KnightHacks](https://knighthacks.org/)| Univeristy of Central Florida, Orlando, FL | March 27-29, 2020 | TBA | Registration is open. ~About 7 hours away from GGC.
+|[PackHacks](https://ncsupackhacks.org/)| NC State University, Raleigh, NC | April 4th-5th, 2020 | TBA | Registration is open. ~Abour 5.5 hours away from GGC.
+|[HealthHack](https://healthtech.hack.gt/)| Georgia Tech, Atlanta, GA | November 3-4, 2020 | TBA | :---
```

Comment:
I think it should be April 3-5 @pchazares1

Reply:
Okay changed HealthHacks date to April 3-5